### DISCUSSION TITLE
non docker installa documentation

### DISCUSSION THREAD

**Discussion by ithakaa** [YoRyan/mailrise#21] [General]
can we get non docker installation documentation, please

**Comment by YoRyan** [comment]
I mention the non-Docker method [here](https://github.com/YoRyan/mailrise#from-pypi): Install the PyPI package, write a configuration file, and pass that file to the executable. I even provide a sample systemd service.

**Reply by ithakaa** [reply]
thanks, where are you hiding the mailrise.conf file ?

**Reply by YoRyan** [reply]
Well, there's the sample configuration:

```yaml
configs:
  pushover:
    urls:
      - pover://[...]
```

Obviously this example is for Pushover. For your own service of choice, you should consult the [documentation](https://github.com/caronc/apprise/wiki/config_yaml) from Apprise.
